yarn-project/end-to-end/src/e2e_p2p/upgrade_governance_proposer.test.ts

```diff
@@ -24,6 +25,8 @@ const BOOT_NODE_UDP_PORT = 45000;
 
 const DATA_DIR = './data/gossip';
 
+jest.setTimeout(1000000);
```

Comment:
unimportant: consistency nazi but in gossip_network this is 1000 * 60 * 10

Reply:
hmm thats made me just realise the DATA_DIR above collides with gossip_network.test.ts